wscript

```diff
@@ -846,7 +846,7 @@ def size_resources(ctx):
     elif ctx.env.MICRO_FAMILY == 'STM32F7':
         max_size = 1024 * 1024
     elif ctx.env.MICRO_FAMILY == 'NRF52840':
-        max_size = 512 * 1024
```

Comment:
probably this should be by device rather than by micro, but that's not this PR's fault

Reply:
yeah, needs to be fixed, this propagated STM32 pattern